client-datastore-cassandra/src/main/resources/schema-cassandra.cql

```diff
@@ -65,3 +65,16 @@ CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.client_session_by_location (
 ) WITH comment='Index Table to look up wireless client sessions by location and equipment used by TIP WLAN CloudSDK. Records automatically expire after 90 days'
     AND default_time_to_live = 7776000;
 
+CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.client_session_by_mac_string (
+    customerId int,
+    locationId bigint,
+    equipmentId bigint,    
+    macAddress bigint,
+    macAddressString text,    
+    
+    PRIMARY KEY (customerId, macAddressString)
+) WITH comment='Index Table to look up wireless client sessions by macAddressString, location, and equipment by TIP WLAN CloudSDK. Records automatically expire after 90 days'
+    AND default_time_to_live = 7776000;
```

Comment:
please change the value to 30 days = 2592000 seconds, all other tables were changed in a separate PR